zerver/management/commands/change_password.py

```diff
@@ -0,0 +1,66 @@
+import getpass
+from argparse import ArgumentParser
+from typing import Any
+
+from django.contrib.auth.password_validation import validate_password
+from django.core.exceptions import ValidationError
+from django.core.management.base import CommandError
+
+from zerver.lib.management import ZulipBaseCommand
+
+
+class Command(ZulipBaseCommand):
+    # This is our version of the original Django changepassword command adjusted
+    # to be able to find UserProfiles by email+realm.
```

Comment:
Can you try to comment more completely on what parts of this are modified from the original?  And could we, e.g., subclass for any parts of this?

Reply:
Added more details. We can't subclass, because it's just one chunk of code in `handle()` in https://github.com/django/django/blob/stable/3.2.x/django/contrib/auth/management/commands/changepassword.py#L34 where we have to replace 
```
        if options['username']:
            username = options['username']
        else:
            username = getpass.getuser()

        try:
            u = UserModel._default_manager.using(options['database']).get(**{
                UserModel.USERNAME_FIELD: username
            })
        except UserModel.DoesNotExist:
            raise CommandError("user '%s' does not exist" % username)
```

with our own way of fetching the user